src/blocks/blocks/form/style.scss

```diff
@@ -62,6 +108,30 @@
 				}
 			}
 
+			&.full-tablet {
+				@media ( max-width: 1024px ) {
+					flex-direction: column;
+					align-items: center;
+
+					.wp-block-button__link {
+						width: 100%;
+						max-width: unset;
+					}
+				}
+			}
+
+			&.full-mobile {
+				@media ( max-width:640px ) {
+					flex-direction: column;
+					align-items: center;
+
+					.wp-block-button__link {
+						width: 100%;
+						max-width: unset;
+					}
+				}
+			}
+
```

Comment:
@HardeepAsrani, I need to use classes for the Full style on submit button alignment. The problem is that I do not know how to integrate this with the Global CSS Default.